tests/test_embeds.py

```diff
@@ -80,6 +80,48 @@ def test_len(embed: Embed) -> None:
     assert len(embed) == 69
 
 
+def test_eq() -> None:
+    embed_1, embed_2 = Embed(), Embed()
+    assert embed_1 == embed_2
+
+    # attribute=MISSING should not affect __eq__ (with attribute i mean any attribute of an Embed object)
+    embed_1.color = Color(123456)
+    assert not embed_1 == embed_2
+
+    embed_1.color = MISSING
+    assert embed_1 == embed_2
+
+    embed_1.add_field(name="This is a test field", value="69 test 69")
+    embed_2.add_field(name="This is a test field", value="69 test 69", inline=False)
+    assert not embed_1 == embed_2
+
+    embed_1.clear_fields(), embed_2.clear_fields()  # type: ignore
+
+    embed_1.title, embed_2.title = MISSING, MISSING
+    assert embed_1 == embed_2
```

Comment:
```suggestion
    embed_1, embed_2 = Embed(), Embed()
```
easier to create new embeds at this point